pkg/updates/devices.go

```diff
@@ -19,12 +19,12 @@ type Inventory struct {
 }
 
 type devices struct {
-	ID                    string        `json:"id"`
-	Ostree                systemProfile `json:"system_profile"`
-	SubscriptionManagerId string        `json:"subscription_manager_id"`
+	ID     string        `json:"id"`
+	Ostree systemProfile `json:"system_profile"`
 }
 
 type systemProfile struct {
+	RhcClientID          string   `json:"rhc_client_id"`
```

Comment:
```suggestion
	RHCClientID          string   `json:"rhc_client_id"`
```

Go best practices coming your way! :D